packages/bpk-component-bottom-sheet/src/BpkBottomSheet-test.tsx

```diff
@@ -85,17 +91,39 @@ describe('BpkBottomSheet', () => {
   it('renders correctly with action props', () => {
     const { asFragment } = render(
       <BpkBottomSheet
-        ariaLabelledby='bottom-sheet'
+        {...props}
         actionText='Action'
-        id="my-bottom-sheet"
-        isOpen
         onAction={jest.fn()}
-        onClose={jest.fn()}
         wide
       >
         Bottom Sheet content
       </BpkBottomSheet>
     );
     expect(asFragment()).toMatchSnapshot();
   });
+  it('renders correctly with ariaLabelledBy prop', () => {
+    const { container } = render(
+      <BpkBottomSheet
+        {...props}
+        ariaLabelledby='my-bottomsheet-title-id'
```

Comment:
it's already part of `props` so I don't think we need it here?

Reply:
Yeah I we could avoid it but I thought it was nice to be explicit when we are asserting based on that property?